schemas/com.snowplowanalytics.snowplow.badrows/schema_violations/jsonschema/1-0-0

```diff
@@ -0,0 +1,303 @@
+                  "properties": {
+                    "errors": {
+                      "type": "array",
+                      "minItems": 1,
+                      "items": {
+                        "properties": {
+                          "path": {
+                            "type": "string",
+                            "description": "JSON Path to an issue in the faulty JSON Schema"
+                          },
+                          "message": {
+                            "type": "string",
+                            "description": "Human-readable message describing the issue with the schema"
+                          }
+                        },
+                        "required": [ "path", "message" ],
+                        "additionalProperties": false
+                      }
+                    }
+                  },
+                  "required": [ "errors" ],
+                  "additionalProperties": false
+                },
+                {
+                  "description": "Invalid data - Schema was resolved and valid, but the given instance is not valid against it. The error comes from a validator",
+                  "properties": {
+                    "errors": {
+                      "type": "array",
+                      "minItems": 1,
+                      "items": {
+                        "properties": {
+                          "message": {
+                            "type": "string",
+                            "description": "Human-readable message describing the problem in the invalidated data"
+                          },
+                          "path": {
+                            "type": "string",
+                            "description": "JSON Path to an issue in the faulty data"
+                          },
+                          "keyword": {
+                            "type": "string",
+                            "description": "JSON Schema keyword that invalidated this instance"
+                          },
+                          "targets": {
+                            "type": "string",
+                            "description": "Violating value"
+                          }
+                        },
+                        "required": [ "message", "path", "keyword", "targets" ],
+                        "additionalProperties": false
+                      }
+                    }
+                  },
+                  "required": [ "errors" ],
+                  "additionalProperties": false
+                }
+              ]
+            }
+          ]
+        }
+      },
+      "required": [ "timestamp", "schemaKey", "messages" ],
+      "additionalProperties": false
+    },
+    "payload": {
+      "type": "object",
+      "description": "The payload that resulted in schema violations",
+      "properties": {
+        "trackerProtocolPayload": {
+          "type": "object",
```

Comment:
Love this @BenFradet - being able to query the top level fields in the trackerProtocolPayload is going to make debugging _much_ easier